R/outbreak_step.R

```diff
@@ -23,29 +23,35 @@
 #' @export
 #'
 #' @examples
-#'
-#'\dontrun{
 #' # incubation period sampling function
-#' incfn <- dist_setup(dist_shape = 2.322737,dist_scale = 6.492272)
+#' incfn <- dist_setup(dist_shape = 2.32, dist_scale = 6.49)
 #' # delay distribution sampling function
-#' delayfn <- dist_setup(1.651524, 4.287786)
+#' delayfn <- dist_setup(dist_shape = 1.65, dist_scale = 4.28)
 #' # generate initial cases
-#' case_data <- outbreak_setup(num.initial.cases = 5,incfn,delayfn,k=1.95,prop.asym=0)
+#' case_data <- outbreak_setup(
+#'   num.initial.cases = 5,
+#'   incfn = incfn,
+#'   delayfn = delayfn,
+#'   k = 1.95,
+#'   prop.asym = 0
+#' )
 #' # generate next generation of cases
-#' case_data <- outbreak_step(case_data = case_data,
-#' disp.iso = 1,
-#' disp.com = 0.16,
-#' r0isolated = 0,
-#' r0subclin = 1.25,
-#' disp.subclin = 0.16,
-#' r0community = 2.5,
-#' prop.asym = 0,
-#' incfn = incfn,
-#' delayfn = delayfn,
-#' prop.ascertain = 0,
-#' k = 1.95,
-#' quarantine = FALSE)[[1]]
-#'}
+#' out <- outbreak_step(
+#'   case_data = case_data,
+#'   disp.iso = 1,
+#'   disp.com = 0.16,
+#'   disp.subclin = 0.16,
+#'   r0isolated = 0,
+#'   r0subclin = 1.25,
+#'   r0community = 2.5,
+#'   prop.asym = 0,
+#'   incfn = incfn,
+#'   delayfn = delayfn,
+#'   prop.ascertain = 0,
+#'   k = 1.95,
+#'   quarantine = FALSE
+#' )
+#' case_data <- out[[1]]
```

Comment:
display case data between steps?

Reply:
I've added printing the `case_data` object in 62bfe1d9ca5faaa44b40a6813ec7f13cd5aa8817.

Reply:
do you also want to emit case_data before the outbreak step call (so line 38ish) - as in, to show what's actually changed?

Reply:
Resolved in 079da980eb98a08f930cc9ad04980a2055bf07a5.